pygmt/datasets/samples.py

```diff
@@ -123,3 +123,38 @@ def load_fractures_compilation():
     fname = which("@fractures_06.txt", download="c")
     data = pd.read_csv(fname, header=None, sep=r"\s+", names=["azimuth", "length"])
     return data[["length", "azimuth"]]
+
+
+def load_hotspots():
+    """
+    Load a table with the locations, names, and suggested icon sizes of
+    hotspots.
+
+    This is the ``@hotspots.txt`` dataset used in the GMT tutorials.
+
+    The data are downloaded to a cache directory (usually ``~/.gmt/cache``) the
+    first time you invoke this function. Afterwards, it will load the data from
+    the cache. So you'll need an internet connection the first time around.
+
+    Returns
+    -------
+    data : pandas.DataFrame
+        The data table. Use ``print(data.describe())`` to see the available
+        columns.
```

Comment:
I know the rest of the sample datasets have the same sentence, but maybe we should just say what the column names are? Something like so:

```suggestion
        The data table with columns "longitude", "latitude", "icon_size", and
        "placename".
```

What do you think?

Reply:
I think it's a good idea!